app.rb

```diff
@@ -0,0 +1,32 @@
+require 'sinatra'
+require 'erb'
+require 'pry-byebug'
+require './helpers/weather_scraper.rb'
+require './helpers/craigslist_scraper.rb'
+
+enable :sessions
+set :server, 'thin'
+
+get '/' do
+  erb :home
+end
+
+post '/home' do
+  session[:location] = params[:location]
+  session[:keywords] = params[:keywords]
```

Comment:
This is great; I did not master the post very well if I am honest; having the post do the work then redirecting to results and posting erb :results is more elegant.